industry_restaurant/data/appointment_type.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="appointment_type_1" model="appointment.type">
+        <field name="name">Floor 1</field>
+        <field name="is_published" eval="True"/>
+        <field name="category">recurring</field>
+        <field name="cover_properties">{"background-image":"url(industry_restaurant/static/src/binary/appointment_resource/table1.jpg)","background_color_class":"o_cc3 o_cc","background_color_style":"","opacity":"0","resize_class":"o_half_screen_height o_record_has_cover","text_align_class":""}</field>
+        <field name="appointment_tz" model="res.users" eval="obj().env.ref('base.user_admin').tz or 'Europe/Brussels'"/>
+        <field name="schedule_based_on">resources</field>
+        <field name="min_cancellation_hours">0.25</field>
+        <field name="resource_total_capacity">4</field>
```

Comment:
This is wrong, but you don't see it because it'll be recomputed as soon as you assign the resources
```suggestion
```